src/OpenIddict.Abstractions/Primitives/OpenIddictExtensions.cs

```diff
@@ -812,7 +812,7 @@ public static string GetClaim([NotNull] this ClaimsPrincipal principal, [NotNull
                 throw new ArgumentException("The claim type cannot be null or empty.", nameof(type));
             }
 
-            return principal.FindFirst(type)?.Value;
+            return principal.FindFirst(claim => string.Equals(claim.Type, type))?.Value;
```

Comment:
Is there any particular reason to use case-sensitive here?

Reply:
There is a test for this ( GetClaim_IsCaseSensitive() ) in old project - that's why I changed testing code. If it should be case-insensitive now - let's remove this test method and revert to "FindFirst(type)"

Reply:
Oh yeah, good point. Can you please revert it and rename the test to `GetClaim_IsCaseInsensitive`?

Reply:
Done